tasks/db_deploy/migrations/migrate_versions_4_to_5/migrate.py

```diff
@@ -6,10 +6,14 @@
 from typing import Dict, List
 
 import orca_shared
-from orca_shared.database.shared_db import get_admin_connection, logger
+from aws_lambda_powertools import Logger
+from orca_shared.database.shared_db import get_admin_connection
 
 import migrations.migrate_versions_4_to_5.migrate_sql as sql
 
+# Set AWS powertools logger
+LOGGER = Logger()
```

Comment:
Recommend reverting here, and adding to card to standardize logger propagation.

Reply:
reverted